Source/Core/Common/MathUtil.h

```diff
@@ -30,6 +31,45 @@ constexpr auto Lerp(const T& x, const T& y, const F& a) -> decltype(x + (y - x)
   return x + (y - x) * a;
 }
 
+template <typename Dest, typename T>
+constexpr Dest SaturatingCast(T value)
+{
+  static_assert(std::is_integral<Dest>());
+
+  constexpr Dest lo = std::numeric_limits<Dest>::lowest();
```

Comment:
Shouldn't these need to be renamed as `LO` and `HI` according to Dolphin's coding style?

Reply:
I think it'd be weird here because there's no constant number on the right-hand side (e.g. `constexpr float PI = 3.1415;`) and it's a constant that varies (oxymoron, I know) depending on the Dest type.

If we followed the rule literally, all constexpr variables would need to be written in all caps -- in constexpr functions, most/all variables would need to be in all caps, which is super ugly and verbose when there's no risk of confusion anyway.

If I recall correctly @lioncash said that the rule should probably be updated to reflect modern C++ so I'm inclined to leaving the variable names as is.